src/api/app/policies/report_policy.rb

```diff
@@ -6,7 +6,7 @@ def show?
     CommentPolicy.new(user, record.reportable).maintainer? if record.reportable_type == 'Comment'
   end
 
-  def create?
+  def create? # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
You should better wrap the whole method with a `# rubocop:disable Metrics/CyclomaticComplexity` and `# rubocop:enable Metrics/CyclomaticComplexity`